dropwizard-e2e/src/main/java/com/example/badlog/BadLogApp.java

```diff
@@ -0,0 +1,25 @@
+package com.example.badlog;
+
+import io.dropwizard.Application;
+import io.dropwizard.Configuration;
+import io.dropwizard.setup.Environment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BadLogApp extends Application<Configuration> {
```

Comment:
I agree that public is not needed, but dropwizard-e2e isn't deployed to maven and it's not imported into other dropwizard modules. The other applications in e2e suffer the same problem that the access is too wide. Maybe another PR is needed to clean up all the unnecessary wide access.